ui/test/unit/appointments/controllers/manage/appointmentsCreateController.spec.js

```diff
@@ -913,5 +913,30 @@ describe("AppointmentsCreateController", function () {
 
         $scope.onKeyDownOnEndTime();
         expect($scope.showEndTimes).toEqual($scope.endTimes);
-    })
+    });
+
+    it('should include appointment provider if the provider is not available in all available providers', function () {
+        var allAvailableProviders = [{name: 'superman', uuid: '1'}, {name: 'mahmoud_h', uuid: '2'}];
+        appointmentContext.appointment = {provider: {name: 'someone', uuid: '3'}};
+        appointmentCreateConfig.providers = allAvailableProviders;
+
```

Comment:
Can you please remove the blank lines?Apart from this,everything looks fine to me